main.cpp

```diff
@@ -279,230 +303,283 @@ struct CommericalAeroplane
-    //provide inflight entertainment
     void provideInflightEntertainment(int passengerSeatNumber, std::string mediaSelction);  
 };
 
-/*
-Thing 5 ) Player
-5 properties:
-    1) body weight (float)
-    2) sponsor (std:string)
-    3) number of games played (int)
-    4) scoring average (float)
-    5) name (std::string)
-3 things it can do:
-    1) run
-    2) score
-    3) train
-*/
+float CommercialAeroplane::Cockpit::increaseAirSpeed(float deltaV, float currentAirSpeed)
+{
+return deltaV + currentAirSpeed;
+}
+
+bool CommercialAeroplane::Cockpit::groundProximityWarningAlarm(bool nearGround, float proximityWarningThreshold)
+{
+    if(nearGround == true && proximityWarningThreshold < 800)
+    groundProximityalarmStateOn = true;
+
+    return groundProximityalarmStateOn;
+}
+bool CommercialAeroplane::Cockpit::autopilotDisengagement(bool pilotAndCopilotChecklistComplete)
+{
+    if (pilotAndCopilotChecklistComplete)
+    autoPilotLightOn = false;
+    return autoPilotLightOn;
+}
+
+void CommercialAeroplane::accelerateForTakeOff(float groundSpeed, bool armDoorsAndCrossCheck, bool runWayClear, bool airspaceClear)
+{
+    if (runWayClear && armDoorsAndCrossCheck && airspaceClear)
+    std::cout << "Clear for take off" << std::endl;
+
+    if (groundSpeed > speedRequiredForTakeOff)
+    landingGearDown = false;
+}
+
+void CommercialAeroplane::transportPeople(int totalPassengers, int totalCrew)
+{
+    std::cout << "Total number on board: " << totalPassengers + totalCrew << std::endl;
+}
+void CommercialAeroplane::provideInflightEntertainment(int passengerSeatNumber, std::string mediaSelction)
+{
+    std::cout << "Passenger " << passengerSeatNumber << " selected " << mediaSelction << std::endl;
+}
 
 struct Player
 {
-    //body weight (float)
     float bodyMassKG = 101.34f;
-    //sponsor (std:string)
     std::string sponsor = "Nike";
-    //number of games played (int)
     int numGamesPlayed = 104;
-    //scoring average (float)
     float scoringAverage = 1.78f;
-    //name (std::string)
     std::string name = "Ronaldo";
 
-    //run
     float run(int howFast, float howFar, bool startWithLeftFoot);
-    //score
     void score(bool hasBall, bool inScoringPosition);
-    //train
     void train(std::string trainingRequired, float fitnessScore);
 };
+float Player::run(int howFast, float howFar, bool startWithLeftFoot)
+{
+    float speed = 0;
+    float energyUsed = 0;
 
-/*
-Thing 6) Coach
-5 properties:
-    1) name (std::string)
-    2) years experience (int)
-    3) trophies won (int)
-    4) salary (double)
-    5) inspiring (bool)
-3 things it can do:
-    1) give motivational speech
-    2) lose job
-    3) give press conference
-*/
+    if (startWithLeftFoot)
+    speed = howFar / howFast;
+    energyUsed = speed * bodyMassKG;
+
+    return energyUsed;
+}
+
+void Player::score(bool hasBall, bool inScoringPosition)
+{
+    if (hasBall && inScoringPosition)
+    std::cout << "Goal scored" << std::endl;
+}
```

Comment:
It is unclear if you want to apply both of these variable assignments when this `if` statement is true.
Why?
- it is lacking indentation
- it is lacking curly braces.

remove the ambiguity by being explicit with what code should execute when the `if` statement is `true`
- [ ] add curly braces
- [ ] revise